src/software/ai/hl/stp/play/kickoff_friendly_play_test.cpp

```diff
@@ -33,13 +36,19 @@ TEST_F(KickoffFriendlyPlayTest, test_kickoff_friendly_play)
         // TODO: Implement proper validation
         // https://github.com/UBC-Thunderbots/Software/issues/1396
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(9.5))
-            {
-                yield();
-            }
+            robotReceivedBall(4, world_ptr, yield);
```

Comment:
We should also check that the ball is kicked right?

Reply:
Added